packages/terra-list/src/MultiSelectList.jsx

```diff
@@ -111,6 +111,18 @@ class MultiSelectList extends React.Component {
   cloneChildItems(items) {
     const disableUnselectedItems = this.state.selectedIndexes.length >= this.validatedMaxCount();
 
+    if (items === []) {
+      return items;
+    }
+
+    if (items.constructor !== Array) {
+      const wrappedOnClick = this.wrappedOnClickForItem(items, 0);
+      const wrappedOnKeyDown = this.wrappedOnKeyDownForItem(items, 0);
+      const newProps = this.newPropsForItem(items, 0, wrappedOnClick, wrappedOnKeyDown);
+
+      return React.cloneElement(items, newProps);
+    }
+
     return items.map((item, index) => {
```

Comment:
Would this do the same thing:
`return React.Children.map(items, (item, index) => {`